lubabaz-dhaka.md

```diff
@@ -0,0 +1,10 @@
+#Dhaka, Bangladesh
+
+##Fun Facts 
+- Known as the city of mosques. 
+
+##Places to Go
+- Sangshad Bhaban
+
+##Places to Avoid
+- Crowdy areas, I guess!
```

Comment:
Aaaa!

Well! Dhaka is an awesome place to visit. During celebrations, you can avoid TSC.